src/components/global/nav-bar/nav-bar.vue

```diff
@@ -0,0 +1,103 @@
+<script>
+    import { defineComponent } from 'vue';
+
+    import MonteCinemaIcon from '../../../assets/icons/monte-cinema-logo.svg';
+    import ToggleNavOpenIcon from '../../../assets/icons/toggle-nav-open-icon.svg';
+    import ToggleNavClosedIcon from '../../../assets/icons/toggle-nav-closed-icon.svg';
+    import NavLinks from './nav-links.vue';
+    import NavButtonContainer from './nav-button-container.vue';
+
+    export default defineComponent({
+        name: 'NavBar',
+        components: {
+            MonteCinemaIcon,
+            ToggleNavOpenIcon,
+            ToggleNavClosedIcon,
+            NavLinks,
+            NavButtonContainer,
+        },
+        props: {
+            userType: {
+                type: String,
+                default: 'public', //Others: client, employee
+            },
+        },
+        data() {
+            return {
+                isScreenSmall: false,
+                isNavOpen: false,
+            }
+        },
+        methods: {
+            watchScreenSmall() {
+                this.isScreenSmall = window.innerWidth < 1024;
+            },
+            toggleNavOpen() {
+                this.isNavOpen = !this.isNavOpen;
+            },
+        },
+        created() {
+            this.watchScreenSmall();
+            window.addEventListener('resize', this.watchScreenSmall);
+        },
+    });
+</script>
+
+<template>
+    <nav 
+        class="navbar"
+        :userType="userType" 
+    >
+        <div class="nav-header-small">
+            <div class="nav-monte-logo">
+                <MonteCinemaIcon />
+            </div>
+            <div 
+                class="nav-toggle-icon" 
+                v-if="isScreenSmall" 
+                @click="toggleNavOpen"
+            >
+                <ToggleNavOpenIcon v-if="!isNavOpen" />
+                <ToggleNavClosedIcon v-if="isNavOpen" />
+            </div>
+        </div>
+        <NavLinks v-if="isScreenSmall && isNavOpen || !isScreenSmall" />
+        <NavButtonContainer v-if="isScreenSmall && isNavOpen || !isScreenSmall" />
```

Comment:
Let's follow the DRY principle and create method with long enough name to describe what it does and reuse it : )

Reply:
That will not be necessary as I  found another - more CSS'y - solution.